packages/tests-postman/README.md

```diff
@@ -16,7 +16,7 @@ All APIs and configs will be tested with Newman/Postman, and additional settings
 
 Install package with:
 ```bash
-npm i
+npm ci
```

Comment:
why the need for ci install here?

Reply:
A clean install here will throw an error if there are any discrepancies between the `package.json` and `package-lock.json` files instead of modifying them to resolve the issues before installing the packages, which seemed more appropriate for the testing environment here.